environment.yml

```diff
@@ -0,0 +1,9 @@
+channels:
```

Comment:
Maybe a stylistic choice - is there a reason you are choosing to have multiple env files per task as opposed to one master env file? Are there major conflicts with dependencies? Just thinking that if you expect a user to rerun some of these QC and filtering steps, then they will need to initiate multiple different envs

Reply:
snakemake automatically manages environments, installing them and activating them for the rules that use them (which is controlled by the `conda:`  directive in the rules). Some of these will have dependency conflicts, but in general I have found that the environments have better performance as a whole when they're kept small (faster installation, faster activation). I suppose it is somewhat stylistic, but I do this across all of my workflows